### ISSUE TITLE
Locked out of MultiBit - Password changed involuntarily, please help!

### ISSUE THREAD

**Issue by Fallace** [Multibit-Legacy/multibit#331]
I am kind of freaking out right now, and could really use some help! To start my problem, a few weeks ago I have bought Bitcoins on localbitcoins.com. Reading into security issues, I thought it would be best to sent them to an electronical wallet. That's why I downloaded multibit, and sent some Bitcoins there some time ago. I've only made 3 transactions in total with it. Today, I have sent some amount to my brother, so he could profit from the rising price of bitcoins as well.
When I made the trasaction (this afternoon), I did not need a password. Today, localbitcoins.com was acting a bit weird (not loading some pages, very long loading times in general), which made me distrust localbitcoins for the amount of money I had in it (400 euros or something) - so I sent all of it to my MultiBit wallet, which is on my desktop. This took a long time (the loading) but it finally came through. After that, I wanted to send a small amount to another friend of mine, and I suddenly needed to use a password. I am 100% sure that I did not need a password when I sent bitcoins to my brother this afternoon, and I am 100% sure I did not change the password in between the two transactions. The really weird thing is that my BTC are not gone from my wallet, they are still in their. However, I can't send anything or change the password. I have good protection software on my desktop (licensed Norton antivirus), so I have no clue what exactly happened. I am also sure that no one has touched my desktop in the meantime.
TL;DR:
Bought Bitcoins on Localbitcoins
Tried to send the remaining BTC on localbitcoins to my MultiBit which has been working perfectly fine
Suddenly has a password and I didn't change it
Could anyone explain what happened? I remember that during the transaction from localbitcoins to my multibit there was a long waiting time and it said (establishing safe network). I know I'm extremely stupid for not backing up my wallet and not having a password on it. I could shoot myself for not protecting it properly, which is totally my own fault. I didn't know MultiBit required a password, and I should definitely have researched it better. Please don't flame me for that, I could really use some help as I have lost a lot of money and am not a very wealthy student. Could anyone explain what happened and if there is a way to retrieve them?

**Comment by Jessica22022** [comment]
@Fallace 
did you recover your multibit wallet,
this is the issue which occurs a lot for older versions of multibit, I can recover your wallet if you still have this problem contact me at : [[walletsrecoveryservice@gmail.com]
